olive/passes/onnx/conversion.py

```diff
@@ -449,3 +449,34 @@ def _convert_distributed_model_on_device(
             model_name_pattern=DistributedOnnxModel.DEFAULT_RANKED_MODEL_NAME_FORMAT,
             num_ranks=world_size,
         )
+
+
+class OnnxOpVersionConversion(Pass):
+    @staticmethod
+    def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigParam]:
+        latest_opset_version = onnx.defs.onnx_opset_version()
+
+        config = {
+            "target_opset": PassConfigParam(
+                type_=int,
+                default_value=latest_opset_version,
+                description="The version of the default (ai.onnx) opset to target. Default: latest opset version.",
+            ),
+        }
+        config.update(get_external_data_config())
+        return config
+
+    def _run_for_config(
+        self, model: ONNXModel, data_root: str, config: Dict[str, Any], output_model_path: str
+    ) -> ONNXModel:
+        # get current models's opset version
+        model_proto = model.load_model()
+        model_opset_version = model_proto.opset_import[0].version
+        if model_opset_version == config["target_opset"]:
+            # we have no good solution to skip the one pass once it is executed, just save the model again to
+            # target output path
+            logger.info(f"Model is already in target opset version {config['target_opset']}.")
+            return model
+        else:
```

Comment:
nit: more of a coding style/preference so no action needed. The code is clearer if 1) line 482 is under the else clause or 2) there is no else clause at all since the if clause returns

I don't think we have a specific preference for 1 or 2 in our code base. There is a mix of both. Personally, I use 2 for cases where the if clause returns.

Reply:
Updated! Thanks!